pages/documentation/simulation/sensitivity/openlf.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+latex: true
+---
+
+# OpenLoadFlow
+
+PowSyBl OpenLoadFlow is an open-source power flow implementation in Java provided by PowSyBl. The source code is hosted on [GitHub](https://github.com/powsybl/powsybl-open-loadflow). 
+
+* TOC
+{:toc}
+
+## DC sensitivity analysis
+
+A DC sensitivity analysis starts from the DC flows computing described in the [power flow section](../powerflow/openlf.md#dc-flows-computing). Simple sensitivity analyses are supported as:
+- How an injection increase of 1 MW will impact the flow of a branch ;
+- How a phase shifting of 1 degree of a phase tap changer will impact the flow of a branch.
+
+This could be done in a set of branches given by the user.
+
+### Sensitivities on the base network 
+
+Injection increases could be either an active power increase of a generator $$targetP$$ or an active power increase of a load $$P0$$. To get the impact of an injection increase in a given bus into a given branch, we compute the right-hand side \\(b\\) corresponding to an injection of 1 MW at this bus and 0 MW elsewhere. The LU decomposition gives the matrices \\(L\\) and \\(U\\) in order to compute the vector \\(\theta\\). Then it is easy to retrieve the active power flow in the branch.
```

Comment:
Should be better to use `$$` everywhere to avoid mixing both syntax? Maybe numbers should be backquoted (`1 MW`, `0 MW`) to increase readability

Reply:
Indeed. I am editing it